### DISCUSSION TITLE
Make the source hex color values of a theme available as css variables.

### DISCUSSION THREAD

**Discussion by stefanobartoletti** [saadeghi/daisyui#3186] [Ideas / feature requests]
I think it would be useful to have the source hex color definitions (i.e. these https://daisyui.com/theme-generator/ ) be available as CSS variables in addition to the generated OKLCH values.

There can be situations where these hex values should be accessed from javascript (one simple example would be to add the Chrome Mobile theme https://developer.chrome.com/blog/support-for-theme-color-in-chrome-39-for-android), but unlike CSS, JS does not have a simple and native way to handle OKLCH values.

**Comment by saadeghi** [comment]
`oklch(var(--b1))` should work (for `base-100` color)
For example `<meta name="theme-color" content="oklch(var(--b1))">`  
I haven't tried on Android, but it works on Safari.

**Comment by stefanobartoletti** [comment]
Ok thanks. Anyway, that was only an example, the main concern is to be able to access the HEX base value from Javascript
